src/pybind/mgr/telemetry/module.py

```diff
@@ -0,0 +1,376 @@
+"""
+Telemetry module for ceph-mgr
+
+Collect statistics from Ceph cluster and send this back to the Ceph project
+when user has opted-in
+"""
+import errno
+import json
+import re
+import requests
+import uuid
+import time
+from datetime import datetime
+from threading import Event
+from collections import defaultdict
+
+from mgr_module import MgrModule
+
+
+class Module(MgrModule):
+    run = False
```

Comment:
i don't really understand why shall we make these variables class variable? as they are not shared among multiple instances of `Module`. and neither are they supposed to outlive the Module instance.

Reply:
Well, that's a personal thing. I think it's easier to find where they have been defined. I could define them in the init() function, but that's not always very clear.

For me it doesn't mind that much where they are defined.